requirements.txt

```diff
@@ -1,6 +1,6 @@
 coloredlogs
 flatbuffers
-numpy >= @Python_NumPy_VERSION@
```

Comment:
in ORT 1.18.0 we produced ort packages which required numpy >= 1.21.6  , so should we make it >= 1.21.6 to stay consistent?

Reply:
Sure. I will update it now.